src/arch/arm/64/head.S

```diff
@@ -23,7 +23,7 @@
 #define CR_ALIGN_CLEAR   0
 #endif
 
-#if defined(CONFIG_ARM_HYPERVISOR_SUPPORT) && defined(CONFIG_AARCH64_USER_CACHE_ENABLE)
+#if !defined(CONFIG_ARM_HYPERVISOR_SUPPORT) && defined(CONFIG_AARCH64_USER_CACHE_ENABLE)
```

Comment:
I think it would help if there is a comment here in the code with what is mentioned in https://github.com/seL4/seL4/pull/1026#discussion_r1224125835. We can't explain all details, but give some hints at least.

Reply:
Yes, it should have a comment explaining.